src/pages/login/index.tsx

```diff
@@ -1,41 +1,53 @@
 import React, { useCallback, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { Center } from '~/components/Box';
 import { getDashboardPath } from '~/constants/paths';
 import { useSelector } from '~/store/hooks';
 import { login } from '~/store/slices/CreateUserSlice';
+import { LoginLayout } from './layout';
+import { LoginForm, formData } from './components/LoginForm';
+import styled from 'styled-components';
```

Comment:
Package imports should be placed at the top of the imports list - I actually though that there is an es-lint rule/plugin configured for it - it probably should be updated

Reply:
BTW is that just for a readibility or it has some other consequences?

Reply:
It's purely for the readability, but helps a lot with organising and reading super-long lists of imports